bootstrap_db.py

```diff
@@ -1,13 +1,17 @@
-import os
-
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy import create_engine
 
 from models import Role, Outcome, Base
 
-def bootstrap(db_host='localhost', db_password='docker'):
-    engine = create_engine('postgresql://postgres:%s@%s:5432' % (db_password, db_host))
+def bootstrap():
+    engine = create_engine(
+        'postgresql+auroradataapi://:@/tintagel',
```

Comment:
Is tintagel the endpoint?

Reply:
It the name of the [database](https://www.postgresql.org/docs/9.0/tutorial-createdb.html). Previously it just was using default name but I gave it a name when I created our serverless instance.

Reply:
Is the name going to be different for dev vs prd?

Reply:
Nope. The name would be the same.